testing/oak_echo_bin/README.md

```diff
@@ -1,4 +1,10 @@
-# Echo Baremetal
+# Oak Echo
 
-Echo baremetal binary that can be run under
+Echo freestanding binary that can be run under
 [crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/).
+
+Echo binary receives a message as bytes, and sends back the same bytes, without interpreting them.
+
+This binary uses IDL to communicate between the Untrusted Launcher and the guest
+VM.
+The IDL is defined here: [`schema.fbs`](testing/oak_echo_runtime/schema.fbs)
```

Comment:
IDL is a language: Interface Definition Language.

What is linked is the interface of the service, not the language itself.

Reply:
Renamed to interface